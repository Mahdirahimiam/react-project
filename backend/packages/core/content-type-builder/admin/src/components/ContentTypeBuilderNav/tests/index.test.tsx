/* eslint-disable check-file/filename-naming-convention */
import { Layout, lightTheme, ThemeProvider } from '@strapi/design-system';
import { render } from '@testing-library/react';
import { createMemoryHistory } from 'history';
import { IntlProvider } from 'react-intl';
import { Router } from 'react-router-dom';

import { ContentTypeBuilderNav } from '../ContentTypeBuilderNav';

import { mockData } from './mockData';

jest.mock('../useContentTypeBuilderMenu.ts', () => {
  return {
    useContentTypeBuilderMenu: jest.fn(() => ({
      menu: mockData,
      searchValue: '',
      onSearchChange() {},
    })),
  };
});

const makeApp = () => {
  const history = createMemoryHistory();

  return (
    <IntlProvider messages={{}} defaultLocale="en" textComponent="span" locale="en">
      <ThemeProvider theme={lightTheme}>
        <Router history={history}>
          <Layout sideNav={<ContentTypeBuilderNav />}>
            <div />
          </Layout>
        </Router>
      </ThemeProvider>
    </IntlProvider>
  );
};

describe('<ContentTypeBuilderNav />', () => {
  it('renders and matches the snapshot', () => {
    const App = makeApp();
    const { container } = render(App);

    expect(container).toMatchSnapshot();
  });
});
