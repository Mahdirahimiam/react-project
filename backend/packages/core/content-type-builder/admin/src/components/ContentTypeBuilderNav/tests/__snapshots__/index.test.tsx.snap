// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ContentTypeBuilderNav /> renders and matches the snapshot 1`] = `
.c10 {
  border: 0;
  -webkit-clip: rect(0 0 0 0);
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c5 {
  font-weight: 600;
  font-size: 1.125rem;
  line-height: 1.22;
  color: #32324d;
}

.c25 {
  font-weight: 600;
  font-size: 0.6875rem;
  line-height: 1.45;
  text-transform: uppercase;
  color: #666687;
}

.c34 {
  font-size: 0.875rem;
  line-height: 1.43;
  color: #32324d;
}

.c41 {
  font-size: 0.75rem;
  line-height: 1.33;
  color: #4945ff;
}

.c47 {
  font-size: 0.875rem;
  line-height: 1.43;
  font-weight: 500;
  color: #32324d;
}

.c2 {
  padding-top: 24px;
  padding-right: 16px;
  padding-bottom: 8px;
  padding-left: 24px;
}

.c6 {
  background: #ffffff;
  padding: 8px;
  border-radius: 4px;
  border-color: #dcdce4;
  border: 1px solid #dcdce4;
  cursor: pointer;
}

.c11 {
  padding-top: 16px;
}

.c12 {
  background: #eaeaef;
}

.c15 {
  padding-top: 8px;
  padding-bottom: 16px;
}

.c18 {
  padding-top: 8px;
  padding-right: 16px;
  padding-bottom: 8px;
  padding-left: 24px;
}

.c20 {
  padding-right: 24px;
  position: relative;
}

.c21 {
  text-align: left;
}

.c24 {
  padding-right: 4px;
}

.c27 {
  background: #eaeaef;
  padding-right: 8px;
  padding-left: 8px;
  position: absolute;
  right: 0;
  top: 50%;
  min-width: 20px;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
}

.c30 {
  background: #f6f6f9;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 32px;
}

.c33 {
  padding-left: 8px;
}

.c35 {
  padding-left: 32px;
}

.c36 {
  background: transparent;
  margin-top: 8px;
}

.c39 {
  color: #666687;
  width: 0.5rem;
  height: 0.5rem;
}

.c42 {
  padding-top: 8px;
  padding-right: 16px;
  padding-bottom: 8px;
  padding-left: 32px;
}

.c48 {
  background: #f6f6f9;
  padding-top: 8px;
  padding-bottom: 8px;
  padding-left: 48px;
}

.c49 {
  padding-bottom: 56px;
}

.c3 {
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c7 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c16 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 8px;
}

.c17 {
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  gap: 4px;
}

.c22 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c28 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c37 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  gap: 8px;
}

.c44 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c38 {
  border: none;
  position: relative;
  outline: none;
}

.c38[aria-disabled='true'] {
  pointer-events: none;
}

.c38[aria-disabled='true'] svg path {
  fill: #666687;
}

.c38 svg path {
  fill: #4945ff;
}

.c38:after {
  -webkit-transition-property: all;
  transition-property: all;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -4px;
  bottom: -4px;
  left: -4px;
  right: -4px;
  border: 2px solid transparent;
}

.c38:focus-visible {
  outline: none;
}

.c38:focus-visible:after {
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -5px;
  bottom: -5px;
  left: -5px;
  right: -5px;
  border: 2px solid #4945ff;
}

.c40 path {
  fill: #666687;
}

.c29 {
  border-radius: 4px;
  height: 1.5rem;
}

.c8 {
  position: relative;
  outline: none;
}

.c8 > svg {
  height: 12px;
  width: 12px;
}

.c8 > svg > g,
.c8 > svg path {
  fill: #ffffff;
}

.c8[aria-disabled='true'] {
  pointer-events: none;
}

.c8:after {
  -webkit-transition-property: all;
  transition-property: all;
  -webkit-transition-duration: 0.2s;
  transition-duration: 0.2s;
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -4px;
  bottom: -4px;
  left: -4px;
  right: -4px;
  border: 2px solid transparent;
}

.c8:focus-visible {
  outline: none;
}

.c8:focus-visible:after {
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -5px;
  bottom: -5px;
  left: -5px;
  right: -5px;
  border: 2px solid #4945ff;
}

.c13 {
  height: 1px;
  border: none;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  margin: 0;
}

.c9 {
  border-color: #dcdce4;
  height: 2rem;
  width: 2rem;
}

.c9 svg g,
.c9 svg path {
  fill: #8e8ea9;
}

.c9:hover svg g,
.c9:focus svg g,
.c9:hover svg path,
.c9:focus svg path {
  fill: #666687;
}

.c9[aria-disabled='true'] svg path {
  fill: #666687;
}

.c0 {
  display: grid;
  grid-template-columns: auto 1fr;
}

.c50 {
  overflow-x: hidden;
}

.c1 {
  width: 14.5rem;
  background: #f6f6f9;
  position: -webkit-sticky;
  position: sticky;
  top: 0;
  height: 100vh;
  overflow-y: auto;
  border-right: 1px solid #dcdce4;
  z-index: 1;
}

.c14 {
  width: 1.5rem;
  background-color: #dcdce4;
}

.c31 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: #32324d;
}

.c31 svg > * {
  fill: #666687;
}

.c31.active {
  background-color: #f0f0ff;
  border-right: 2px solid #4945ff;
}

.c31.active svg > * {
  fill: #271fe0;
}

.c31.active .c4 {
  color: #271fe0;
  font-weight: 500;
}

.c31:focus-visible {
  outline-offset: -2px;
}

.c32 {
  width: 0.75rem;
  height: 0.25rem;
}

.c32 * {
  fill: #666687;
}

.c43 svg {
  height: 0.25rem;
}

.c43 svg path {
  fill: #4a4a6a;
}

.c45 {
  border: none;
  padding: 0;
  background: transparent;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c46 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 0.75rem;
  -webkit-transform: rotateX(0deg);
  -ms-transform: rotateX(0deg);
  transform: rotateX(0deg);
}

.c23 {
  border: none;
  padding: 0;
  background: transparent;
}

.c26 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-transform: rotateX(0deg);
  -ms-transform: rotateX(0deg);
  transform: rotateX(0deg);
}

.c19 svg {
  height: 0.25rem;
}

.c19 svg path {
  fill: #8e8ea9;
}

<div>
  <div
    class="c0"
  >
    <nav
      aria-label="Content-Types Builder"
      class="c1"
    >
      <div
        class="c2"
      >
        <div
          class="c3"
        >
          <h2
            class="c4 c5"
          >
            Content-Types Builder
          </h2>
          <span>
            <button
              aria-disabled="false"
              aria-labelledby=":r1:"
              class="c6 c7 c8 c9"
              tabindex="0"
              type="button"
            >
              <span
                class="c10"
              >
                Search
              </span>
              <svg
                aria-hidden="true"
                fill="none"
                focusable="false"
                height="1rem"
                viewBox="0 0 24 24"
                width="1rem"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  clip-rule="evenodd"
                  d="m23.813 20.163-5.3-5.367a9.792 9.792 0 0 0 1.312-4.867C19.825 4.455 15.375 0 9.913 0 4.45 0 0 4.455 0 9.929c0 5.473 4.45 9.928 9.912 9.928a9.757 9.757 0 0 0 5.007-1.4l5.275 5.35a.634.634 0 0 0 .913 0l2.706-2.737a.641.641 0 0 0 0-.907ZM9.91 3.867c3.338 0 6.05 2.718 6.05 6.061s-2.712 6.061-6.05 6.061c-3.337 0-6.05-2.718-6.05-6.06 0-3.344 2.713-6.062 6.05-6.062Z"
                  fill="#32324D"
                  fill-rule="evenodd"
                />
              </svg>
            </button>
          </span>
        </div>
        <div
          class="c11"
        >
          <hr
            class="c12 c13 c14"
          />
        </div>
      </div>
      <div
        class="c15"
      >
        <ol
          class="c16"
        >
          <li>
            <div
              class="c17"
            >
              <div
                class="c18 c19"
              >
                <div
                  class="c20"
                >
                  <button
                    aria-controls=":r3:"
                    aria-expanded="true"
                    class="c21 c22 c23"
                  >
                    <div
                      class="c24"
                    >
                      <span
                        class="c4 c25"
                      >
                        Collection Types
                      </span>
                    </div>
                    <div
                      class="c26"
                    >
                      <svg
                        aria-hidden="true"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 14 8"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M14 .889a.86.86 0 0 1-.26.625L7.615 7.736A.834.834 0 0 1 7 8a.834.834 0 0 1-.615-.264L.26 1.514A.861.861 0 0 1 0 .889c0-.24.087-.45.26-.625A.834.834 0 0 1 .875 0h12.25c.237 0 .442.088.615.264a.86.86 0 0 1 .26.625Z"
                          fill="#32324D"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </div>
                  </button>
                  <div
                    class="c27 c28 c29"
                    transform="translateY(-50%)"
                  >
                    <span
                      class="c4 c25"
                    >
                      2
                    </span>
                  </div>
                </div>
              </div>
              <ol
                id=":r3:"
              >
                <li>
                  <a
                    class="c30 c31"
                    href="/plugins/content-type-builder/content-types/application::address.address"
                  >
                    <div
                      class="c22"
                    >
                      <svg
                        class="c32"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 4 4"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <rect
                          fill="#A5A5BA"
                          height="4"
                          rx="2"
                          width="4"
                        />
                      </svg>
                      <div
                        class="c33"
                      >
                        <span
                          class="c4 c34"
                        >
                          Address
                        </span>
                      </div>
                    </div>
                  </a>
                </li>
                <li>
                  <a
                    class="c30 c31"
                    href="/plugins/content-type-builder/content-types/application::category.category"
                  >
                    <div
                      class="c22"
                    >
                      <svg
                        class="c32"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 4 4"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <rect
                          fill="#A5A5BA"
                          height="4"
                          rx="2"
                          width="4"
                        />
                      </svg>
                      <div
                        class="c33"
                      >
                        <span
                          class="c4 c34"
                        >
                          Category
                        </span>
                      </div>
                    </div>
                  </a>
                </li>
              </ol>
            </div>
            <div
              class="c35"
            >
              <button
                aria-disabled="false"
                class="c36 c37 c38"
                type="button"
              >
                <svg
                  class="c39 c40"
                  fill="none"
                  height="1rem"
                  viewBox="0 0 24 24"
                  width="1rem"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M24 13.604a.3.3 0 0 1-.3.3h-9.795V23.7a.3.3 0 0 1-.3.3h-3.21a.3.3 0 0 1-.3-.3v-9.795H.3a.3.3 0 0 1-.3-.3v-3.21a.3.3 0 0 1 .3-.3h9.795V.3a.3.3 0 0 1 .3-.3h3.21a.3.3 0 0 1 .3.3v9.795H23.7a.3.3 0 0 1 .3.3v3.21Z"
                    fill="#212134"
                  />
                </svg>
                <span
                  class="c4 c41"
                >
                  Create new collection type
                </span>
              </button>
            </div>
          </li>
          <li>
            <div
              class="c17"
            >
              <div
                class="c18 c19"
              >
                <div
                  class="c20"
                >
                  <button
                    aria-controls=":r4:"
                    aria-expanded="true"
                    class="c21 c22 c23"
                  >
                    <div
                      class="c24"
                    >
                      <span
                        class="c4 c25"
                      >
                        Single Types
                      </span>
                    </div>
                    <div
                      class="c26"
                    >
                      <svg
                        aria-hidden="true"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 14 8"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M14 .889a.86.86 0 0 1-.26.625L7.615 7.736A.834.834 0 0 1 7 8a.834.834 0 0 1-.615-.264L.26 1.514A.861.861 0 0 1 0 .889c0-.24.087-.45.26-.625A.834.834 0 0 1 .875 0h12.25c.237 0 .442.088.615.264a.86.86 0 0 1 .26.625Z"
                          fill="#32324D"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </div>
                  </button>
                  <div
                    class="c27 c28 c29"
                    transform="translateY(-50%)"
                  >
                    <span
                      class="c4 c25"
                    >
                      1
                    </span>
                  </div>
                </div>
              </div>
              <ol
                id=":r4:"
              >
                <li>
                  <a
                    class="c30 c31"
                    href="/plugins/content-type-builder/content-types/application::homepage.homepage"
                  >
                    <div
                      class="c22"
                    >
                      <svg
                        class="c32"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 4 4"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <rect
                          fill="#A5A5BA"
                          height="4"
                          rx="2"
                          width="4"
                        />
                      </svg>
                      <div
                        class="c33"
                      >
                        <span
                          class="c4 c34"
                        >
                          Homepage
                        </span>
                      </div>
                    </div>
                  </a>
                </li>
              </ol>
            </div>
            <div
              class="c35"
            >
              <button
                aria-disabled="false"
                class="c36 c37 c38"
                type="button"
              >
                <svg
                  class="c39 c40"
                  fill="none"
                  height="1rem"
                  viewBox="0 0 24 24"
                  width="1rem"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M24 13.604a.3.3 0 0 1-.3.3h-9.795V23.7a.3.3 0 0 1-.3.3h-3.21a.3.3 0 0 1-.3-.3v-9.795H.3a.3.3 0 0 1-.3-.3v-3.21a.3.3 0 0 1 .3-.3h9.795V.3a.3.3 0 0 1 .3-.3h3.21a.3.3 0 0 1 .3.3v9.795H23.7a.3.3 0 0 1 .3.3v3.21Z"
                    fill="#212134"
                  />
                </svg>
                <span
                  class="c4 c41"
                >
                  Create new single type
                </span>
              </button>
            </div>
          </li>
          <li>
            <div
              class="c17"
            >
              <div
                class="c18 c19"
              >
                <div
                  class="c20"
                >
                  <button
                    aria-controls=":r5:"
                    aria-expanded="true"
                    class="c21 c22 c23"
                  >
                    <div
                      class="c24"
                    >
                      <span
                        class="c4 c25"
                      >
                        Components
                      </span>
                    </div>
                    <div
                      class="c26"
                    >
                      <svg
                        aria-hidden="true"
                        fill="none"
                        height="1rem"
                        viewBox="0 0 14 8"
                        width="1rem"
                        xmlns="http://www.w3.org/2000/svg"
                      >
                        <path
                          clip-rule="evenodd"
                          d="M14 .889a.86.86 0 0 1-.26.625L7.615 7.736A.834.834 0 0 1 7 8a.834.834 0 0 1-.615-.264L.26 1.514A.861.861 0 0 1 0 .889c0-.24.087-.45.26-.625A.834.834 0 0 1 .875 0h12.25c.237 0 .442.088.615.264a.86.86 0 0 1 .26.625Z"
                          fill="#32324D"
                          fill-rule="evenodd"
                        />
                      </svg>
                    </div>
                  </button>
                  <div
                    class="c27 c28 c29"
                    transform="translateY(-50%)"
                  >
                    <span
                      class="c4 c25"
                    >
                      2
                    </span>
                  </div>
                </div>
              </div>
              <ol
                id=":r5:"
              >
                <li>
                  <div
                    class=""
                  >
                    <div
                      class="c42 c43"
                    >
                      <div
                        class="c44"
                      >
                        <button
                          aria-controls=":r6:"
                          aria-expanded="true"
                          class="c45"
                        >
                          <div
                            class="c46"
                          >
                            <svg
                              aria-hidden="true"
                              fill="none"
                              height="1rem"
                              viewBox="0 0 14 8"
                              width="1rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                clip-rule="evenodd"
                                d="M14 .889a.86.86 0 0 1-.26.625L7.615 7.736A.834.834 0 0 1 7 8a.834.834 0 0 1-.615-.264L.26 1.514A.861.861 0 0 1 0 .889c0-.24.087-.45.26-.625A.834.834 0 0 1 .875 0h12.25c.237 0 .442.088.615.264a.86.86 0 0 1 .26.625Z"
                                fill="#32324D"
                                fill-rule="evenodd"
                              />
                            </svg>
                          </div>
                          <div
                            class="c33"
                          >
                            <span
                              class="c4 c47"
                            >
                              Basic
                            </span>
                          </div>
                        </button>
                      </div>
                    </div>
                    <ul
                      id=":r6:"
                    >
                      <li>
                        <a
                          class="c48 c31"
                          href="/plugins/content-type-builder/component-categories/basic/basic.simple"
                        >
                          <div
                            class="c22"
                          >
                            <svg
                              class="c32"
                              fill="none"
                              height="1rem"
                              viewBox="0 0 4 4"
                              width="1rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <rect
                                fill="#A5A5BA"
                                height="4"
                                rx="2"
                                width="4"
                              />
                            </svg>
                            <div
                              class="c33"
                            >
                              <span
                                class="c4 c34"
                              >
                                Simple
                              </span>
                            </div>
                          </div>
                        </a>
                      </li>
                    </ul>
                  </div>
                </li>
                <li>
                  <div
                    class=""
                  >
                    <div
                      class="c42 c43"
                    >
                      <div
                        class="c44"
                      >
                        <button
                          aria-controls=":r7:"
                          aria-expanded="true"
                          class="c45"
                        >
                          <div
                            class="c46"
                          >
                            <svg
                              aria-hidden="true"
                              fill="none"
                              height="1rem"
                              viewBox="0 0 14 8"
                              width="1rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                clip-rule="evenodd"
                                d="M14 .889a.86.86 0 0 1-.26.625L7.615 7.736A.834.834 0 0 1 7 8a.834.834 0 0 1-.615-.264L.26 1.514A.861.861 0 0 1 0 .889c0-.24.087-.45.26-.625A.834.834 0 0 1 .875 0h12.25c.237 0 .442.088.615.264a.86.86 0 0 1 .26.625Z"
                                fill="#32324D"
                                fill-rule="evenodd"
                              />
                            </svg>
                          </div>
                          <div
                            class="c33"
                          >
                            <span
                              class="c4 c47"
                            >
                              Default
                            </span>
                          </div>
                        </button>
                      </div>
                    </div>
                    <ul
                      id=":r7:"
                    >
                      <li>
                        <a
                          class="c48 c31"
                          href="/plugins/content-type-builder/component-categories/default/default.closingperiod"
                        >
                          <div
                            class="c22"
                          >
                            <svg
                              class="c32"
                              fill="none"
                              height="1rem"
                              viewBox="0 0 4 4"
                              width="1rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <rect
                                fill="#A5A5BA"
                                height="4"
                                rx="2"
                                width="4"
                              />
                            </svg>
                            <div
                              class="c33"
                            >
                              <span
                                class="c4 c34"
                              >
                                Closingperiod
                              </span>
                            </div>
                          </div>
                        </a>
                      </li>
                      <li>
                        <a
                          class="c48 c31"
                          href="/plugins/content-type-builder/component-categories/default/default.dish"
                        >
                          <div
                            class="c22"
                          >
                            <svg
                              class="c32"
                              fill="none"
                              height="1rem"
                              viewBox="0 0 4 4"
                              width="1rem"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <rect
                                fill="#A5A5BA"
                                height="4"
                                rx="2"
                                width="4"
                              />
                            </svg>
                            <div
                              class="c33"
                            >
                              <span
                                class="c4 c34"
                              >
                                Dish
                              </span>
                            </div>
                          </div>
                        </a>
                      </li>
                    </ul>
                  </div>
                </li>
              </ol>
            </div>
            <div
              class="c35"
            >
              <button
                aria-disabled="false"
                class="c36 c37 c38"
                type="button"
              >
                <svg
                  class="c39 c40"
                  fill="none"
                  height="1rem"
                  viewBox="0 0 24 24"
                  width="1rem"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M24 13.604a.3.3 0 0 1-.3.3h-9.795V23.7a.3.3 0 0 1-.3.3h-3.21a.3.3 0 0 1-.3-.3v-9.795H.3a.3.3 0 0 1-.3-.3v-3.21a.3.3 0 0 1 .3-.3h9.795V.3a.3.3 0 0 1 .3-.3h3.21a.3.3 0 0 1 .3.3v9.795H23.7a.3.3 0 0 1 .3.3v3.21Z"
                    fill="#212134"
                  />
                </svg>
                <span
                  class="c4 c41"
                >
                  Create a new component
                </span>
              </button>
            </div>
          </li>
        </ol>
      </div>
    </nav>
    <div
      class="c49 c50"
    >
      <div />
    </div>
  </div>
  <div
    class="c10"
  >
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-log"
      role="log"
    />
    <p
      aria-live="polite"
      aria-relevant="all"
      id="live-region-status"
      role="status"
    />
    <p
      aria-live="assertive"
      aria-relevant="all"
      id="live-region-alert"
      role="alert"
    />
  </div>
</div>
`;
